CMakeLists.txt

```diff
@@ -102,6 +104,11 @@ else()
   set(BUILD_PROTOBUF ON)
 endif()
 
+if (NOT BUILD_PYTHON)
+  set(BUILD_SHM_BINDINGS OFF)
```

Comment:
Maybe it should depend on backend?

Reply:
What do you mean by backend?

Now it depends on the python bindings for consistency (why compile a C++ module that is only used in Python bindings), and I turn it off for qnx.

Reply:
I thought about using `cmake_dependent_option`, but `if()` should do as well.

Reply:
Everything is already a dependent option, I didn't want to put more logic there.

Reply:
I don't like it in the slightest. Whether or not we build python bindings should not affect how we build core.
It should be a part of QNX config.

Reply:
Removed.